ngraph/frontend/paddlepaddle/src/model.cpp

```diff
@@ -502,6 +453,142 @@ namespace ngraph
             m_tensor_values[name] = constant;
         }
 
+        Place::Ptr InputModelPDPD::InputModelPDPDImpl::addOutput(Place::Ptr place)
+        {
+            auto tensor_place = pdpd::castToTensorPlace(place);
+            if (!tensor_place->is_output())
+            {
+                m_outputs.push_back(place);
+                return tensor_place;
+            }
+            FRONT_END_THROW("Place is already output.");
+        }
+
+        void InputModelPDPD::InputModelPDPDImpl::removeOutput(Place::Ptr place)
+        {
+            auto tensor_place = pdpd::castToTensorPlace(place);
+            m_outputs.erase(std::remove(m_outputs.begin(), m_outputs.end(), tensor_place),
+                            m_outputs.end());
+            m_graph_changed = true;
+        }
+
+        void InputModelPDPD::InputModelPDPDImpl::cutAndAddNewOutput(
+            Place::Ptr place, const std::string& new_name_optional)
+        {
+            auto tensor_place = pdpd::castToTensorPlace(place);
+            if (tensor_place)
+            {
+                // Find outputs that are connected to the cut place. These target outputs should be
+                // cut off.
+                std::vector<Place::Ptr> target_out_tensor;
+                auto callback_out = [&](const std::shared_ptr<OpPlacePDPD>& op_place,
```

Comment:
`op_place` is not used. Is it also 'reserved for future possible use'?

Reply:
it's used in determine_cut_nodes() function